src/resolver/didValidator.ts

```diff
@@ -52,9 +53,40 @@ const resolverInstance = new Resolver(didWeb.getResolver())
  * DID document metadata, and trust validation outcome.
  */
 export async function resolve(did: string, options: ResolverConfig): Promise<TrustResolution> {
+  if (!options.didResolver) {
+    options.didResolver = getCredoTsDidResolver(options.agentContext)
+  }
   return await _resolve(did, options)
 }
 
+/**
+ * Creates a DID Resolver instance that uses the Credo-TS internal `DidResolverService`
+ * to resolve Decentralized Identifiers (DIDs).
+ *
+ * This resolver delegates all resolution requests to the `DidResolverService` registered
+ * within the provided `AgentContext`.
+ *
+ * @param agentContext - The agent context containing the global operational state
+ * of the agent, including registered services, modules, DIDs, wallets, storage, and configuration
+ * from Credo-TS.
+ *
+ * @returns A `did-resolver` `Resolver` instance configured to use Credo-TS for DID resolution.
+ */
+function getCredoTsDidResolver(agentContext: AgentContext): Resolver {
+  const didResolverService = agentContext.dependencyManager.resolve(DidResolverService)
```

Comment:
```suggestion
  const didResolverApi = agentContext.dependencyManager.resolve(DidsApi)
```
Pretty much the same, but it is usually more reliable to get the API because it is meant to be more stable than an internal component such as the `DidResolverService`.